roles/config-rh-sso/tasks/prereq.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: 'Pre-Req: Install required packages'
+  package:
+    name: '{{ item }}'
+    state: installed
+  with_items:
+  - firewalld
+  - python-firewall
+
+- name: 'Start firewalld service'
+  service:
+    name: firewalld
+    enabled: yes
+    state: started
+
+- name: 'Pre-Req: Open Firewall for Red Hat SSO'
+  firewalld:
+    port: '{{ item }}'
+    permanent: yes
+    state: enabled
+    immediate: yes
+  with_items:
+  - 8080/tcp
+  - 9990/tcp
+
+- name: 'Register service public IPv4'
+  os_server_facts:
+    server: at1*
+  register: external_ip.ansible_facts.openstack_servers[0].public_v4
```

Comment:
This is OpenStack specific. Let's look for a way to avoid this as the role should be IaaS agnostic + there should be a way to do this at a (dynamic) inventory and/or playbook level.

Reply:
Yeah, because I wanted to have the service up and running, I tackled this approach which is, as you pointed out, OpenStack specific. I am working to generalize this part.

Reply:
Solved.